verifyFiles.py

```diff
@@ -207,7 +209,7 @@ def read_checksum_from_file(checksums_after_file_path):
          for line in file:
 	     if (len(line.split(' ', 1)) == 2) and '/' in line:
 		 filename=((line.split(' ', 1)[1]).strip())
-		 filename=filename.split('/', 1)[1]
+		 filename=filename.split('/', 2)[2]
```

Comment:
not sure why you're taking the second element here. Is there any assumption on how the file name looks like?